components/layout/fragment_tree/box_fragment.rs

```diff
@@ -67,6 +67,24 @@ pub(crate) struct BlockLevelLayoutInfo {
     pub block_margins_collapsed_with_children: CollapsedBlockMargins,
 }
 
+#[derive(MallocSizeOf)]
+pub(crate) struct BoxFragmentRareData {
+    /// Information that is specific to a layout system (e.g., grid, table, etc.).
+    pub specific_layout_info: Option<SpecificLayoutInfo>,
```

Comment:
Wasn't the plan to also move `scroll_node_id` into here?

Reply:
Yes, it would be possible after the https://github.com/servo/servo/pull/37871 is merged.

Reply:
I suppose we are going to store scroll node ids for scroll containers only. @stevennovaryo could you confirm?

Reply:
Yes, and I believe there is no reason to store `scroll_node_id` for a `overflow: auto` node without overflowing children.

Reply:
Currently, `scroll_node_id` is stored for all box fragments, so I'm not sure if that's an accurate description of how the code works now. I'm not sure how you can only store scroll nodes for scroll containers, when the value is used to determine the owning scroll node of every fragment in the spatial tree.

Reply:
The idea is the, scroll tree node is only relevant (would change the display of the nodes) for specific nodes (e.g., one with transform, scrollable with overflowing child, and sticky). For the other nodes we could find the closest parent where the scroll frame is relevant.

Reply:
Fragments do not have a handle to their parent, so how will you find the containing scroll node?